packages/mongo-nav/src/helpers/Icons.tsx

```diff
@@ -1,20 +1,149 @@
 import React from 'react';
-import { css } from '@leafygreen-ui/emotion';
+import { css, cx } from '@leafygreen-ui/emotion';
 import { HTMLElementProps } from '@leafygreen-ui/lib';
 
 type ProductIconProps = HTMLElementProps<'svg'> & {
-  active: boolean;
+  active?: boolean;
+  variant?: 'small' | 'default';
 };
 
-export function AtlasIcon({ active, ...rest }: ProductIconProps) {
+export function CloudManagerIcon({
+  variant = 'default',
+  active = false,
+  className,
+  ...rest
+}: ProductIconProps) {
+  const size = variant === 'small' ? 10 : 18;
+
+  return (
+    <svg
+      {...rest}
+      width={`${size}px`}
+      height={`${size}px`}
+      viewBox={`0 0 18 18`}
+      className={cx(
+        css`
+          flex-shrink: 0;
+        `,
+        className,
+      )}
+    >
+      <defs>
+        <linearGradient
+          id="cloud-manager-linear-gradient"
+          x1="0.96"
+          y1="15.06"
+          x2="15.02"
+          y2="14.8"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop offset="0" stopColor="#0d6149" />
+          <stop offset="0.37" stopColor="#04a74f" />
+          <stop offset="0.65" stopColor="#00cd57" />
+          <stop offset="0.92" stopColor="#5fd590" />
+          <stop offset="1" stopColor="#7ed8a2" />
+        </linearGradient>
+        <linearGradient
+          id="cloud-manager-linear-gradient-2"
+          x1="-2.54"
+          y1="9"
+          x2="13.27"
+          y2="9"
+          xlinkHref="#cloud-manager-linear-gradient"
+        />
+        <linearGradient
+          id="cloud-manager-linear-gradient-3"
+          x1="21.41"
+          y1="15.32"
+          x2="1.11"
+          y2="15.32"
+          xlinkHref="#cloud-manager-linear-gradient"
+        />
+        <linearGradient
+          id="cloud-manager-linear-gradient-4"
+          x1="19.24"
+          y1="9"
+          x2="-3.78"
+          y2="9"
+          xlinkHref="#cloud-manager-linear-gradient"
+        />
+      </defs>
+      <path
+        className={css`
+          fill: ${variant === 'small'
+            ? '#00804b'
+            : active
```

Comment:
Hmm... I'd think we'd also want to render the logo as gray when it's small and not active, right?

Reply:
Nope, per the spec there's always a color: https://mongodb.invisionapp.com/d/main/#/console/19026513/396615702/preview

Reply:
In this case, that's true. We may want to be able to show a monochromatic version of the smaller logo in future cases though.